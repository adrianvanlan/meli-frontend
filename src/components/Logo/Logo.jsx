import { Link } from 'react-router-dom';
import logo1x from '../../assets/Logo_ML.png';
import logo2x from '../../assets/Logo_ML@2x.png.png';
import './Logo.module.scss';

const logo = ({ width }) => (
  <Link to="/" className="Logo__Link">
    <img
      width={width || 106}
      src={logo1x}
      srcSet={`${logo1x} 53w, ${logo2x} 106w`}
      alt="Logo Mercadolibre"
    />
  </Link>
);

export default logo;
