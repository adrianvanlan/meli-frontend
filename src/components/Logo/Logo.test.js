import Logo from './Logo';
import { MemoryRouter } from 'react-router-dom';
import { render } from '@testing-library/react';

describe('Should render Logo', () => {
  it('should render Logo', () => {
    const wrapper = render(
      <MemoryRouter>
        <Logo />
      </MemoryRouter>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
