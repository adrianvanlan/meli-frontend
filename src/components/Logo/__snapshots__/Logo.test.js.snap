// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Logo should render Logo 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <a
        class="Logo__Link"
        href="/"
      >
        <img
          alt="Logo Mercadolibre"
          src="Logo_ML.png"
          srcset="Logo_ML.png 53w, Logo_ML@2x.png.png 106w"
          width="106"
        />
      </a>
    </div>
  </body>,
  "container": <div>
    <a
      class="Logo__Link"
      href="/"
    >
      <img
        alt="Logo Mercadolibre"
        src="Logo_ML.png"
        srcset="Logo_ML.png 53w, Logo_ML@2x.png.png 106w"
        width="106"
      />
    </a>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
